nodebuilder/header/module_test.go

```diff
@@ -36,8 +36,13 @@ func TestConstructModule_StoreParams(t *testing.T) {
 
 	app := fxtest.New(t,
 		fx.Supply(modp2p.Private),
-		fx.Provide(func() datastore.Batching {
-			return datastore.NewMapDatastore()
+		fx.Supply(modp2p.Bootstrappers{}),
+		fx.Provide(context.Background),
+		fx.Provide(libp2p.New),
+		fx.Provide(conngater.NewBasicConnectionGater),
+		fx.Provide(func() (datastore.Batching, datastore.Datastore) {
+			ds := datastore.NewMapDatastore()
+			return ds, ds
 		}),
```

Comment:
FYI - not sure how this test worked without this to begin with..

Reply:
Does it work without the change?